### ISSUE TITLE
[BUG] `parse_binpat_str` expected at least 4 arguments

### ISSUE THREAD

**Issue by loveraven42** [VoidSec/DriverBuddyReloaded#18]
**Describe the bug**
When I try to decode cdrom.sys, it will occur python warning.

**To Reproduce**
1. use ida to reverse cdrom.sys
2. Ctrl+Alt+A
3. 
```Failed while executing plugin_t.run():
Traceback (most recent call last):
  File "C:/Users/raven/Desktop/ida77sp1/x64_idapronw_hexarm64w_hexarmw_hexmipsw_hexppc64w_hexppcw_hexx64w_hexx86w_220118/plugins/DriverBuddyReloaded.py", line 466, in run
    driver_type = utils.get_driver_id(driver_entry_addr, log_file)
  File "C:/Users/raven/Desktop/ida77sp1/x64_idapronw_hexarm64w_hexarmw_hexmipsw_hexppc64w_hexppcw_hexx64w_hexx86w_220118/plugins\DriverBuddyReloaded\utils.py", line 205, in get_driver_id
    populate_wdf()
  File "C:/Users/raven/Desktop/ida77sp1/x64_idapronw_hexarm64w_hexarmw_hexmipsw_hexppc64w_hexppcw_hexx64w_hexx86w_220118/plugins\DriverBuddyReloaded\wdf.py", line 753, in populate_wdf
    idx = ida_bytes.bin_search(ea, idaapi.BADADDR, ida_bytes.parse_binpat_str("KmdfLibrary"),
  File "C:\Users\raven\Desktop\ida77sp1\x64_idapronw_hexarm64w_hexarmw_hexmipsw_hexppc64w_hexppcw_hexx64w_hexx86w_220118\python\3\ida_bytes.py", line 3903, in parse_binpat_str
    return _ida_bytes.parse_binpat_str(*args)
TypeError: parse_binpat_str expected at least 4 arguments, got 1
```

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS and version: Windows 11
 - IDA version IDA 7.7
 - DriverBuddyReloaded Version: HEad
 - Python Version 3.9

**Additional context**

**Comment by VoidSec** [comment]
Hi, thank you for the issue. It has been caused by the following fix: https://github.com/VoidSec/DriverBuddyReloaded/commit/84598ca94f43653231bc93649ae4b97628500e25#r71017059

Could you please try the release v.1.3 https://github.com/VoidSec/DriverBuddyReloaded/releases/tag/1.3 and let me know the result?

Regards

**Comment by loveraven42** [comment]
you are right, it has been fixed in releases 1.3

**Comment by VoidSec** [comment]
fixed in https://github.com/VoidSec/DriverBuddyReloaded/pull/20
